packages/ui/cypress/tests/watched-accounts.cy.ts

```diff
@@ -186,4 +186,39 @@ describe('Watched Accounts', () => {
     multisigPage.accountHeader().should('be.visible')
     multisigPage.newTransactionButton().should('not.exist')
   })
+
+  it('can not utilize wallet connect when only a watched account', () => {
+    cy.visit(settingsPageWatchAccountUrl)
+    addWatchAccount(
+      watchMultisigs['multisig-with-pure'].pureAddress,
+      watchMultisigs['multisig-with-pure'].name
+    )
+    settingsPage.expandWalletConnectButton().click()
+    settingsPage
+      .walletConnectAlert()
+      .should('be.visible')
+      .should(
+        'contain.text',
+        'Please first select a Pure or Multisig that you are part of, to be able to use it with WalletConnect'
+      )
+    settingsPage.walletConnectKeyInput().should('be.disabled')
+    settingsPage.connectDappButton().should('be.disabled')
+  })
+
+  it('can see but not interact with txs when only a watched account', () => {
```

Comment:
Same
```suggestion
  it('can see but not interact with txs with only watched accounts', () => {
```

Reply:
Updated to "can see but not interact with txs when in watched account mode"